core/src/main/resources/fabric8/default-kind-filename-mapping.adoc

```diff
@@ -0,0 +1,138 @@
+|===
```

Comment:
As this file will be updated when a new resource is to be added, we should add an asciidoc comment here to explain what can be update how.

* don't add extra newlines before the new column
* E.g. no multiline columns are allowed (each column must be an extra line)
* Keep Kind, filenames in a sorted order

etc.